x-pack/plugins/infra/common/constants.ts

```diff
@@ -9,3 +9,5 @@ export const DEFAULT_SOURCE_ID = 'default';
 export const METRICS_INDEX_PATTERN = 'metrics-*,metricbeat-*';
 export const LOGS_INDEX_PATTERN = 'logs-*,filebeat-*,kibana_sample_data_logs*';
 export const TIMESTAMP_FIELD = '@timestamp';
+export const METRICS_APP = 'metrics';
```

Comment:
Does it make sense to create constants for our apps?

Reply:
Sure, this seems like a good idea. We have been talking about utilising constants more recently. One word of warning (which might not apply here necessarily as this is more about running the right branch of code) but the official ID of the metrics app is actually `infrastructure` (https://github.com/elastic/kibana/blob/master/x-pack/plugins/infra/server/features.ts#L15), and this has led to bugs in the past where, for example, the alerting framework has registered things under `metrics` which isn't the ID.

Reply:
Good to know!